portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,33 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+// Fetches comments from the server and adds them to the DOM.
+function LoadComments(){
+  var max=getMax();
+  fetch('/data?user-choice='+max).then(response => response.json()).then((comments) => {
+  const commentListElement=document.getElementById('comment-list');
+  commentListElement.innerHTML=' ';
+  comments.forEach((comment)=>{
+  commentListElement.appendChild(createListElement(comment));
+    });
+  });
+}
+
+// Creates an element that represents a comment
+function createListElement(comment){
+  const commentListElement=document.createElement('li');
+  commentListElement.innerText=comment.comment
+  return commentListElement;
+}
+
+function getMax(){
+  var temp=document.getElementById("maxwant");
+  var max=temp.options[temp.selectedIndex].value;
+  return max;
+}
+
+function deleteComment(){
+  fetch("/delete-list", {method: 'POST'}).then(()=>{
```

Comment:
I think it's fine to just do `fetch("/delete-list", {method: 'POST'});` (without adding the `then...` bit of code).  Since nothing happens in the then block it's fine to leave it out

Reply:
Addressed in latest commit.